pytest.ini

```diff
@@ -1,5 +1,3 @@
 [pytest]
 markers =
     integration: marks integration tests that require source/destination api+app keys.
-env =
-    MAX_WORKERS=1
```

Comment:
Don't we need this for `pytest-recording` ?

Reply:
Using tox now to set this and run tests so no need for this extra setting + dep

Reply:
Yes, just thought maybe we can keep it for safety, it's up to you.

Reply:
Ill omit for now and. reintroduce if needed later on